### ISSUE TITLE
Can I avoid lazy initializers for static fields?

### ISSUE THREAD

**Issue by thw0rted** [cincheo/jsweet#431]
I have some static fields that are constants, but include operators (on literal operands only) in their assignment:

    public static final int HEADER_SIZE =
        1 + // offset
        4 + // payload size
        2 + // flags
        ...

This compiles to Typescript that includes `static HEADER_SIZE` and also `public static HEADER_SIZE_$LI$()` which does the lazy initialization, and all references to HEADER_SIZE are replaced by calls to this method.  I understand the motivation but the code looks ugly and in my particular case it's definitely not needed.

Maybe there's an annotation I could throw on there that tells the source generation not to bother with lazy initialization, and just initialize at eval-time?

----

Oh, also, I just noticed that while actual literal assignments generally don't create a `$LI$` function, I'm seeing them when the literal is negative -- that is, when I have

````
public static final int MAX_VALUE = 100;
public static final int MIN_VALUE = -100;
````

the MAX outputs as `MAX_VALUE : number = 100;` but the MIN outputs with a `$LI$` function. Maybe that should be a separate bug?

**Comment by lgrignon** [comment]
It could be nice indeed, but clearly low priority. That being said, it might be pretty easy to develop if you want to give a try with a PR

**Comment by thw0rted** [comment]
Aha! I've been working off of the quickstart, which is still based on the 2.0.0 release.  There are big changes here (As you saw, I was bit by #377, for example).  My second issue above, about negative value initializers, is no longer a problem, as your tests indicate.  Thanks!

My first issue would still be nice to have, though.  If I have some time in the next few weeks to dig into the transpiler architecture I might try to submit a PR but at the moment I'm OK with manually fixing the output.

**Comment by lgrignon** [comment]
Fixed in 3de2c5ef
